run-qa-checks

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -e
+
+echo ''
+echo 'Checking documentation build status'
+./runsphinx-build
```

Comment:
run this and the broken link checks as last

Reply:
Okay thanks. I am adding a comment to make it clear why openwisp-utils checks are performed first